lib/rbi/printer.rb

```diff
@@ -670,7 +670,15 @@ def sig_modifiers
 
     sig { params(v: Printer).void }
     def print_as_line(v)
-      v.printt("sig { ")
+      sig_modifier = if is_final
+        "(:final)"
+      else
+        ""
+      end
+
+      v.printt("sig")
+      v.print(sig_modifier)
```

Comment:
Can we do this instead?

```rb
v.print("(:final)") if is_final
```

Reply:
Haha. Yeah, makes sense. I think I was thinking that there could different modifier parameters, but it turns out `:final` is the only one anyway.